combust.devel.conf

```diff
@@ -0,0 +1,64 @@
+perl  = /usr/bin/perl
+
+# SEE: combust.conf.sample and combust.docker.conf for full options
+
+# the servername for the combust instance
+servername = combust.local
+
+# default port is 8225
+port = 8230
+
+# external_port.  Set to a non-zero value to set the port number used
+# for external redirects (301's, 302's)
+external_port = 0
+
+# Configure sites
+[www]
+servername = wwwperl.local
+
+
+[qa]
+servername = qaperl.local
+googleanalytics = UA-xxx-4
```

Comment:
I think you can leave out Google analytics and static_base here, as they're not used in the common case.